.github/workflows/test.yml

```diff
@@ -8,7 +8,13 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    name: "Unit Test: ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
```

Comment:
Nice, I'm glad it turned out to be this easy. I'm not worried about WSL but would be nice to at least know the path for supporting that if we really need to (e.g. if WSL Ubuntu turns out in the future to be somehow different than just vanilla Ubuntu). However (and I realize this is still marked as draft) would still like to make sure we include Powershell testing support too.

Reply:
To get the tests running in WSL, we'd need to setup and maintain a self hosted runner (a VM with Windows installed) to pick up the test jobs. [Example of this here](https://github.com/ubuntu/wsl-actions-example/tree/main). We could run the tests in WSL1 natively with Actions, as [documented here](https://github.com/Vampire/setup-wsl) but WSL1 is deprecated and won't be representative of what people would be using the CLI in.

Yup, Powershell testing support has been added in this PR 👍